recipe/meta.yaml

```diff
@@ -1,29 +1,61 @@
-# Switch the version of this recipe based on the VC_VERSION environment variable.
-{% set VERSION = os.environ.get('VC_VERSION', '9')|string %}
-{% set runtime_years = {'9': '2008', '10': '2010', '14': '2015'} %}
-{% set runtime_year = runtime_years[VERSION] %}
+# version numbering is explained at https://stackoverflow.com/questions/42730478/version-numbers-for-visual-studio-2017-boost-and-cmake
+#     This version number follows the "platform toolset"
+{% set vcver="14" %}
+{% set vsyear="2015" %}
+{% set fullver="14.0.25420" %}
 
 package:
-  name: vc
-  version: {{ VERSION }}
+  name: vs{{ vsyear }}
+  version: {{ fullver }}
 
 build:
-  number: 0
-  track_features:
-    - vc{{ VERSION }}
-  skip: true  # [not win]
+  number: 11
+  skip: True  [not win]
 
-requirements:
-  run:
-    - vs{{ runtime_year }}_runtime
+outputs:
+  - name: vs{{ vsyear }}_{{ cross_compiler_target_platform }}
+    script: install_activate.bat
+    track_features:
+      - vc{{vcver}}
+    run_exports:
+      strong:
+        # compatible within major version.  This is MSFT's incrementing of the UCRT version.  It was
+        #    14.0 with VS2015, and 14.1 with VS2017 - if the major version changes, we're assuming
+        #    that binary compatibility breaks.
+        - {{ pin_subpackage('vc') }}
+        - {{ pin_subpackage('vs' ~ vsyear ~ '_runtime', max_pin=None) }}
```

Comment:
Should this be
```suggestion
        - {{ pin_subpackage('vs' ~ vsyear ~ '_runtime') }}
```
?

Reply:
Or does it just not matter due since all `vsyear` ones are compatible anyway?

Reply:
It's `vs2015_runtime` and has the year there. So there's no need.

Reply:
Okay, thanks for confirming.